### DISCUSSION TITLE
CDHSample.plotTreeMap() error

### DISCUSSION THREAD

**Discussion by jonathangebru** [pegasystems/pega-datascientist-tools#145] [Q&A] [answered]
I'm going through your example Example ADM analysis for Pega data science tools. When running CDHSample.plotTreeMap(), gives back an attribute error. 'DataFrame' object has no attribute 'append'. I'm following the steps as it is, so I don't know what is going wrong.

**Comment by StijnKas** [comment, marked as answer]
Hi @jonathangebru, could you please open up an issue for this, and copy over the stack trace showing the full details of this error? As is right now, this looks like a plotly/pandas issue, but it could also be related to polars. As this sounds like a bug, an issue is more appropriate - we can make sure to close it!

**Comment by jonathangebru** [comment]
Hi Stijn, 

Upgrading plotly solved this issue. Thanks for the prompt response!
